worker/dbaccessor/worker_test.go

```diff
@@ -4,15 +4,21 @@
 package dbaccessor
 
 import (
-	"database/sql"
+	"context"
+	sql "database/sql"
```

Comment:
This is redundant right?

Reply:
This is just the formatter being the formatter :facepalm: